examples/fish_automatic_cd.fish

```diff
@@ -0,0 +1,15 @@
+# Automatically change the directory in fish after closing ranger
+#
+# This is a fish alias to automatically change the directory to the last visited
+# one after ranger quits.
+# To undo the effect of this function, you can type "cd -" to return to the
+# original directory.
+#
+# Note: funcsave save the alias in fish's config files, you do not need to copy
+# this file anywhere, just execute it once
+
+alias ranger-cd 'ranger --choosedir=$HOME/.rangerdir; set RANGERDIR (cat $HOME/.rangerdir); cd $RANGERDIR'
```

Comment:
A fully fledged function would be more justified here. It seems there is no cleanup of the rangerdir file either.

Reply:
While I agree from a syntax pov, note that `alias` is just an alias for `function` basically : )

Reply:
I know.  All the more reason to use the actual function syntax.

Reply:
:+1: I changed to a function and added the cleanup.